client/packages.go

```diff
@@ -91,6 +91,8 @@ type Snap struct {
 	GatingHold *time.Time `json:"gating-hold,omitempty"`
 	// if RefreshInhibit is nil, then there is no pending refresh.
 	RefreshInhibit *SnapRefreshInhibit `json:"refresh-inhibit,omitempty"`
+
+	Comps []Component `json:"components,omitempty"`
```

Comment:
I would call this Components,  we don't seem to use .\Comps anywhere else

Reply:
Changed